ember-cli-build.js

```diff
@@ -4,14 +4,9 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function (defaults) {
   const app = new EmberAddon(defaults, {
-    fingerprint: {
-      exclude: ['assets/images']
-    },
-    'ember-cli-favicon': {
-      faviconsConfig: {
-        path: '/ADDON_DOCS_ROOT_URL'
-      }
-    }
```

Comment:
I don't think we should remove the `fingerprint` and `ember-cli-favicon` options. These are important for the docs app.